google-cloud-firestore/src/test/java/com/google/cloud/firestore/QueryTest.java

```diff
@@ -957,20 +957,19 @@ public void retriesAfterRetryableError() throws Exception {
     final boolean[] returnError = new boolean[] {true};
 
     doAnswer(
-            (Answer<RunQueryResponse>)
-                invocation -> {
-                  if (returnError[0]) {
-                    returnError[0] = false;
-                    return queryResponse(
-                            FirestoreException.forServerRejection(
-                                Status.DEADLINE_EXCEEDED, "Simulated test failure"),
-                            DOCUMENT_NAME + "1",
-                            DOCUMENT_NAME + "2")
-                        .answer(invocation);
-                  } else {
-                    return queryResponse(DOCUMENT_NAME + "3").answer(invocation);
-                  }
-                })
+            invocation -> {
+              if (returnError[0]) {
+                returnError[0] = false;
+                return queryResponse(
+                        FirestoreException.forServerRejection(
+                            Status.DEADLINE_EXCEEDED, "Simulated test failure"),
+                        DOCUMENT_NAME + "1",
+                        DOCUMENT_NAME + "2")
+                    .answer(invocation);
+              } else {
+                return queryResponse(DOCUMENT_NAME + "3").answer(invocation);
+              }
+            })
```

Comment:
FYI: Here and below - this is only a formatting change as I removed the unused cast.